openmdao/recorders/hdf5_recorder.py

```diff
@@ -10,6 +10,8 @@
 from openmdao.recorders.base_recorder import BaseRecorder
 from openmdao.util.record_util import format_iteration_coordinate
 
+om_case_version = 1
```

Comment:
Right, sqlite and hdf5 will have different version numbers. Thanks for catching that typo!